tinygrad/tensor.py

```diff
@@ -211,6 +211,11 @@ def full_like(self, fill_value:Scalar, **kwargs):
   def zeros_like(self, **kwargs): return self.full_like(0, **kwargs)
   def ones_like(self, **kwargs): return self.full_like(1, **kwargs)
 
+  @staticmethod
```

Comment:
why staticmethod?

Reply:
I was going by pytorch, where its only in torch.nn.functional, so a static method. Do you want it as a non-static method or have both versions?

Reply:
there are so many functions only in `torch.nn.functional` that is not static in tinygrad (see test_ops). definitely not both. why is one better than another?

Reply:
I agree, seems like the precedent is that the methods that create initialize a new tensor are static, but the ones that modify an existing tensor are not. Changed it to be a regular instance method.